tests/test_tradingcalendar.py

```diff
@@ -26,7 +26,6 @@
 
 class TestTradingCalendar(TestCase):
 
-    @nottest
```

Comment:
Hopefully for BMF and TSE everything will be green but currently LSE only has days defined and ignores trading times (e.g. it has no get_early_closes, get_open_and_closes....).